### ISSUE TITLE
[Feature Request] Game save management

### ISSUE THREAD

**Issue by Shoegzer** [xemu-project/xemu#139]
This feature would allow export/import of game saves from the Xbox HD via the xemu UI.  This might ease the process of transferring the data to and from an actual Xbox console, and permit an easy way to archive save game data for future reference.

In the meantime, a brief explanation of how to reference and extract game save data from the HD, including its location, would certainly be welcome.

**Comment by mborgerson** [comment]
Thanks. I do agree that it can be confusing and cumbersome for people to migrate their game save data, and also their dashboard. Perhaps we can create a migration assistant tool that allows people to easily transfer their Xbox dashboard and game saves to an xemu instance.

**Comment by Shoegzer** [comment]
Hey thanks.  Could there also be an option to migrate a single game's save data as well?  I was thinking more of that, though an option to transfer a full set of saves would also be helpful.

**Comment by Stixil** [comment]
Bump, would love to see the ability to manage game saves without the complexity of the disk image

**Comment by awsms** [comment]
Bumping this, as the current way to do it is very unpractical.

**Comment by Unknow0059** [comment]
> Bumping this, as the current way to do it is very unpractical.

What is the current way? I just wanna backup my saves.

**Comment by mborgerson** [comment]
Just connect with an ftp client and backup E:/

**Comment by awsms** [comment]
> Just connect with an ftp client and backup E:/

Don't you need a custom dashboard for it though? Afaik the only way to get saves without it is by mounting the disk image

**Comment by mborgerson** [comment]
> Don't you need a custom dashboard for it though?

Yes, you can use this open source one https://github.com/Ryzee119/LithiumX/releases

Just launch the ISO in xemu and follow the FTP instructions on the xemu website

**Comment by awsms** [comment]
> Yes, you can use this open source one https://github.com/Ryzee119/LithiumX/releases
> 
> Just launch the ISO in xemu and follow the FTP instructions on the xemu website

Hey thanks for the link. Does LithiumX pops up a ftp server once loaded? I can't find an option for it
edit: can't get it to work with lftp or filezilla. Getting "Failed to retrieve directory listing" errors
edit2: worked after rebooting xemu a couple of times
